### PR TITLE
Bug 1314564 Add Note to Scale EFK Pods When Changes are Made

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1314564

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL 🌂

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -654,6 +654,53 @@ $ oc label nodes --all logging-infra-fluentd=true
 
 endif::openshift-origin[]
 
+[Note]
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: This is a lot of content for a note box, so I would consider taking it out

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -654,6 +654,53 @@ $ oc label nodes --all logging-infra-fluentd=true
...
+====
+When you make changes to any part of the EFK stack, specifically Elasticsearch
+or Fluentd, you should first scale Elasicsearch down to zero and scale Fluentd
+so it doesn not match any other nodes. Then make the changes, and then scale them
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: does not

Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: Then, make the changes and scale Elasicsearch and Fluentd back.

**Comment by ahardin-rh** [comment]
@bmcelvee Just a few comments from me 🎆

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating Container Logs]
|Added to instructions to scale EFK pods when changes are made in the
xref:../install_config/aggregate_logging.adoc#aggregate-logging-understanding-the-deployment[Understanding and Adjusting the Deployment] section.
%

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -654,6 +654,50 @@ $ oc label nodes --all logging-infra-fluentd=true
 
 endif::openshift-origin[]
 
+When you make changes to any part of the EFK stack, specifically Elasticsearch
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: Why is this section under the title _Fluentd_? It covers more than just that tool.

**Comment by mburke5678** [comment]
@bmcelvee A few nits. Otherwise LGTM. In the _rev history_, the first sentence seems awkward. Is that first _to_ extraneous. Can it be removed as in:
Added instructions to scale EFK pods...

**Pull request merged**